stable/mlflow/templates/database.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: postgresql.org/v1
```

Comment:
This needs to be optional, so let's add a condition around it, something like this:

```suggestion
{{ if .Values.mflow.postgres.PostgresDatabaseEnabled -}}
apiVersion: postgresql.org/v1
```

Reply:
Good point, thanks :)!